### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by naoi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,19 @@
 # Cloud Orchestrator
 
-[Cloud Orchestrator](https://cloud.airgap.host) is a Drupal cloud management distribution tailored for cloud administrators and operators.
+[Cloud Orchestrator](https://cloud.airgap.host) is a Drupal cloud management
+distribution tailored for cloud administrators and operators.
 
-The distribution manages resources in public and private clouds. Version 5.x includes support for Amazon EC2 and Amazon VPC/Networking, Kubernetes, OpenStack, and Terraform.
+The distribution manages resources in public and private clouds. Version 6.x
+includes support for Amazon EC2 and Amazon VPC/Networking, Kubernetes,
+OpenStack, and Terraform.
```
Comment (baldwinlouie) [inline README.md]: Let's add VMware too.

**Review by baldwinlouie — CHANGES_REQUESTED** [review]
@naoi I made a small comment.  Please take a look.

File: README.md (hunk 1 above)
Comment (naoi) [reply README.md]: @baldwinlouie Thank you for your review.  I added `VMware`.  Can you please check the patch again?

**Review by baldwinlouie — APPROVED** [review]
Looks good to me.

**Pull request merged**